ui/threejs/orbitDisplay.js

```diff
@@ -92,12 +92,17 @@ export const orbitDisplay = async (
     render();
   }).observe(page);
 
-  const updateGeometry = async (geometry) => {
+  const updateGeometry = async (
+    geometry,
+    { withGrid = true, fit = true} = {}
```

Comment:
This exposes withGrid and fit in updateGeometry. I tried to make it so that it wouldn't impact the way preAlpha works,  but I'm not 100% sure that I did that correctly so it's worth checking that I didn't break something here.

Reply:
Looks good

Reply:
Can you make the spacing happy above?

Reply:
Did I fix the spacing in the next commit for lint?